modules/node_modules/@ncigdc/routes/AnnotationsRoute.js

```diff
@@ -5,25 +5,30 @@ import React from 'react';
 import Relay from 'react-relay';
 import { connect } from 'react-redux';
 import { parse } from 'query-string';
+import JSURL from 'jsurl';
 
 import { handleStateChange } from '@ncigdc/dux/relayProgress';
 import AnnotationsPage from '@ncigdc/containers/AnnotationsPage';
 import { parseIntParam, parseFilterParam } from '@ncigdc/utils/uri';
 
 import { viewerQuery } from './queries';
 
+const DEFAULT_ANNOTATIONS_SORT = null;
+
 class AnnotationsRoute extends Relay.Route {
   static routeName = 'AnnotationsRoute';
 
   static queries = viewerQuery;
 
   static prepareParams = ({ location: { search } }) => {
     const q = parse(search);
+    const annotationsSort = q.annotations_sort ? JSURL.parse(q.annotations_sort) : DEFAULT_ANNOTATIONS_SORT;
+
     return ({
       offset: parseIntParam(q.offset, 0),
       size: parseIntParam(q.size, 20),
       filters: parseFilterParam(q.filters, null),
-      annotations_sort: q.annotations_sort || null,
+      annotations_sort: annotationsSort,
```

Comment:
`parseFilterParam(q.sort, null)`?

parseFilter just runs it through `jsurl.parse`